### PR TITLE
[feat] tools-v2: add scan-state

### PR DESCRIPTION
**Pull request by montaguelhz**
### What problem does this PR solve?

Issue Number: #2360 

Problem Summary:

support bs set scan command in [curve tool]

### What is changed and how it works?

What's Changed:

1.  add tools-v2/pkg/cli/command/curvebs/update/scan_state.go
2. modify tools-v2/pkg/cli/command/curvebs/update.go
3. modify tools-v2/pkg/config/bs.go
4. modify tools-v2/README.md

How it Works:

```
Usage:  curve bs update scan-state [flags]

enable/disable scan for logical pool

Flags:
  -c, --conf string            config file (default is $HOME/.curve/curve.yaml or /etc/curve/curve.yaml)
  -f, --format string          output format (json|plain) (default "plain")
  -h, --help                   print help
      --logicalpoolid uint32   logical pool id[required]
      --mdsaddr string         mds address, should be like 127.0.0.1:6700,127.0.0.1:6701,127.0.0.1:6702
      --rpcretrytimes int32    rpc retry times (default 1)
      --rpctimeout duration    rpc timeout (default 10s)
      --scan                   enable/disable scan for logical pool (default true)
      --showerror              display all errors in command
      --verbose                show some log

Examples:
$ curve bs update scan-state --logicalpoolid 1 [--scan=true/false]
```
command result
```
curve bs update scan-state --logicalpoolid 1 --scan=true
+----+------+---------+--------+
| ID | SCAN | RESULT  | REASON |
+----+------+---------+--------+
| 1  | true | success | null   |
+----+------+---------+--------+
curve bs update scan-state --logicalpoolid 2 --scan=false
+----+-------+--------+---------------------+
| ID | SCAN  | RESULT |       REASON        |
+----+-------+--------+---------------------+
| 2  | false | fail   | LogicalPoolNotFound |
+----+-------+--------+---------------------+
```
Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: tools-v2/pkg/cli/command/curvebs/update/scan_state/scan_state.go [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+	return NewUpdateScanStateCommand().Cmd
+}
+
+func (sCmd *ScanStateCommand) AddFlags() {
+	config.AddBsMdsFlagOption(sCmd.Cmd)
+	config.AddRpcRetryTimesFlag(sCmd.Cmd)
+	config.AddRpcTimeoutFlag(sCmd.Cmd)
+
+	config.AddBSLogicalPoolIdRequiredFlag(sCmd.Cmd)
+	config.AddBsScanOptionFlag(sCmd.Cmd)
```
Comment (Cyber-SiKu) [inline tools-v2/pkg/cli/command/curvebs/update/scan_state/scan_state.go:81]: Flag scan is required, will it be better?

File: tools-v2/pkg/cli/command/curvebs/update/scan_state/scan_state.go [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+	if err.TypeCode() != cmderror.CODE_SUCCESS {
+		return err.ToError()
+	}
+	sCmd.Response = result.(*topology.SetLogicalPoolScanStateResponse)
+	if *sCmd.Response.StatusCode != int32(statuscode.TopoStatusCode_Success) {
+		out[cobrautil.ROW_RESULT] = "fail"
+		out[cobrautil.ROW_REASON] = statuscode.TopoStatusCode_name[*sCmd.Response.StatusCode]
+	} else {
+		out[cobrautil.ROW_RESULT] = "success"
+		out[cobrautil.ROW_REASON] = "null"
```
Comment (Cyber-SiKu) [inline tools-v2/pkg/cli/command/curvebs/update/scan_state/scan_state.go:138]: ditto

File: tools-v2/pkg/cli/command/curvebs/update/scan_state/scan_state.go (hunk 1 above)
Comment (montaguelhz) [reply tools-v2/pkg/cli/command/curvebs/update/scan_state/scan_state.go:81]: In my opinion, from the user's point of view, it is more intuitive for the scan flag to default to true. At the same time, reducing parameters can reduce the user's memory burden.

**Comment by Cyber-SiKu** [comment]
cicheck

**Review by Cyber-SiKu — APPROVED** [review]

**Comment by Cyber-SiKu** [comment]
Please merge your commits into one

**Comment by Cyber-SiKu** [comment]
cicheck

**Review by SeanHai — APPROVED** [review]

**Pull request merged**